gui/default/syncthing/core/syncthingController.js

```diff
@@ -1402,6 +1402,11 @@ angular.module('syncthing.core')
                 autoNormalize: true
             };
             $scope.editingExisting = false;
+            $scope.newIgnores = {
+                edited: false,
```

Comment:
As I said elsewhere. We should probably clean up .stignore and .stfolder as we remove folders, in which circumstances this would never exist. Also, wouldn't the getIgnores backpopulate whats on disk if there is anything?